setup.cfg

```diff
@@ -24,3 +24,12 @@ filterwarnings =
     ignore:::docutils[.*]
     ignore:::jinja2[.*]
     ignore:::yaml[.*]
+
+[mypy]
+python_version = 3.8
+
+[mypy-configobj.*]
+ignore_missing_imports = True
+
+[mypy-docutils.*]
+ignore_missing_imports = True
```

Comment:
typeshed has an unfinished set of type annotations in types-docutils. There's also another set of type annotations available in a different package docutils-stubs. Neither of these have all the bits we need. So for now, we'll ignore typing here.